global.css

```diff
@@ -1,3 +1,124 @@
+:root {
+  --white: hsl(0, 0%, 100%);
+  --white-alpha-75: hsla(0, 0%, 100%, 0.75);
+  --gray-lightness-10: hsl(0, 0%, 10%);
+  --gray-lightness-20: hsl(0, 0%, 20%);
+  --gray-lightness-24: hsl(0, 0%, 24%);
+  --gray-lightness-25: hsl(0, 0%, 25%);
+  --gray-lightness-27: hsl(0, 0%, 27%);
+  --gray-lightness-30: hsl(0, 0%, 30%);
+  --gray-lightness-33: hsl(0, 0%, 33%);
+  --gray-lightness-38: hsl(0, 0%, 38%);
+  --gray-lightness-40: hsl(0, 0%, 40%);
+  --gray-lightness-45: hsl(0, 0%, 45%);
+  --gray-lightness-46: hsl(0, 0%, 46%);
+  --truegray: hsl(0, 0%, 50%);
+  --gray-lightness-60: hsl(0, 0%, 60%);
+  --gray-lightness-65: hsl(0, 0%, 65%);
+  --gray-lightness-66: hsl(0, 0%, 66%);
+  --gray-lightness-73: hsl(0, 0%, 73%);
+  --gray-lightness-76: hsl(0, 0%, 76%);
+  --gray-lightness-80: hsl(0, 0%, 80%);
+  --gray-lightness-84: hsl(0, 0%, 84%);
+  --gray-lightness-85: hsl(0, 0%, 85%);
+  --gray-lightness-86: hsl(0, 0%, 86%);
+  --gray-lightness-87: hsl(0, 0%, 87%);
+  --gray-lightness-88: hsl(0, 0%, 88%);
+  --gray-lightness-90: hsl(0, 0%, 90%);
+  --main-bg: hsl(0, 0%, 90%);
+  --gray-lightness-92: hsl(0, 0%, 92%);
+  --gray-lightness-93: hsl(0, 0%, 93%);
+  --gray-lightness-95: hsl(0, 0%, 95%);
```

Comment:
Do we really need 26 levels of gray? I think a bunch of these could be combined.

Also I don't think "normal" humans can discern the differences in alpha channel between `.05`. `.06`, `.07` and `.1`. Can we combine these as well? I think every 10% is okay, but not anything less than 5%.

Reply:
`.06` makes a difference when stacked on top of another to achieve a combination shade. There's also a very discernible difference between `.05` and `.07` on elements like zebra-striping.

Reply:
I actually did combine quite a few shades of gray. You could probably eliminate at least a few more, but when you're converting the massive volume of values, you can't really go hunting for every element to check if a slightly different shade is detrimental.